build-system/check-package-manager.js

```diff
@@ -27,6 +29,27 @@ function yellow(text) {return '\x1b[33m' + text + '\x1b[0m';}
 /**
  * @fileoverview Makes sure that packages are being installed via yarn
  */
+
+function getNodeLatestLtsMajorVersion() {
+  const schedule = getStdout('curl -L ' + nodeScheduleUrl).trim();
```

Comment:
I read recently that `curl` was made available on Windows 10, are we certain this works well on Windows development devices?

Should this instead use `request`? Or `await fetch()`?

Reply:
@kristoferbaxter excellent point. The restriction here is that we cannot use any component that needs to be installed via `yarn` or `npm`. This is because we run this as a pre-requisite to the first-time install. This is why we can only use stuff that is native to `nodejs` or the OS.

I'll see if I can get my hands on a windows device before merging this.

Reply:
Makes sense @rsimha. Thanks for taking a look.

Reply:
Ideally, we'd have relied on the output of `nvm version-remote --lts` to trivially get the latest LTS version, but this is definitely not available on Windows.

Reply:
@kristoferbaxter I removed the use of `curl` and replaced it with `https.get`. PTAL.